### PR TITLE
Phoenix

### PR DESCRIPTION
**Pull request by tombeynon**

### CODE REVIEW

File: phoenix/chain.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "terra2",
```
Comment (tombeynon) [inline phoenix/chain.json]: This should match the folder name (I think `terra2` makes most sense, so changing the directory name)

File: phoenix/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "phoenix-1",
```
Comment (tombeynon) [inline phoenix/assetlist.json]: This needs to be `chain_name` now, and should match the value in chain.json and the directory name.

File: phoenix/chain.json (hunk 1 above)
Comment (Galadrin) [reply phoenix/chain.json]: done in b6e7a91

File: phoenix/assetlist.json (hunk 2 above)
Comment (Galadrin) [reply phoenix/assetlist.json]: done in b6e7a91

**Pull request closed without merging**